config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'carts/index'
```

Comment:
no need for carts index and make a member route or use proper convention when declaring non resourceful route